### DISCUSSION TITLE
How to use clang build with -static-libstdc++

### DISCUSSION THREAD

**Discussion by XiaBei-cy** [termux/termux-app#3806] [Q&A] [answered]
When I use it, I get an error.
![IMG_20240127_214454](https://github.com/termux/termux-app/assets/76262556/02053c1c-568b-4d04-b09b-445443f7ca00)

**Comment by sk0kanik** [comment, marked as answer]
`-static-libstdc++` should work after installing `ndk-multilib` package

**Reply by XiaBei-cy** [reply]
success, tysm
